### ISSUE TITLE
Unexpected behavior with inheritance and -Dtest with Surefire

### ISSUE THREAD

**Issue by mibutec** [junit-team/junit5#1406]
Having such a construct

```java
public abstract class SomeAbstract5Test {
    @Test
    public void abstr() {
        System.out.println("abstr");
    }
}

public class SomeConcrete5Test extends SomeAbstract5Test {
    @Test
    public void concrete() {
        System.out.println("concrete");
    }
}
```

When executing tests with `mvn test -Dtest=SomeConcrete5Test` only tests defined in concrete test class are executed. The expected behavior (and the behavior provided by JUnit 4) would be to execute all tests inherited, too.

Is there a way to achieve this using JUnit Platform 1.2 and JUnit Jupiter 5.2?

## Related Issues

- #1237
- #1499

**Comment by mibutec** [comment]
Seems to be related to https://github.com/junit-team/junit5/pull/1237

**Comment by mibutec** [comment]
As far I can see

```
<configuration>
  <includes>
    <include>${include.tests}</include>
  </includes>
</configuration>
```
and `mvn test -Dinclude.tests=SomeConcrete5Test` seems to do it. However, expected behavior in native call would be nice.

**Comment by sbrannen** [comment]
Thanks for raising the issue.

We'll look into it.

**Comment by sbrannen** [comment]
Tentatively slated for 5.3 M1 for further investigation.

**Comment by sbrannen** [comment]
This has been fixed in 8856b48a950ca05a19491d8e59d92962bba0917c by @lutovich.

**Comment by sbrannen** [comment]
For an explanation of the cause of this bug, please see the analysis in #1499.

**Comment by sbrannen** [comment]
Reopening this issue to ensure that we don't forget to document the fix in the Release Notes for 5.3.

**Comment by sbrannen** [comment]
Documented in Release Notes in cb779c06522546ead6e684f10a0821716657dc2d
